app/src/main/java/com/radu/newsreader/model/newslist/NewsListFragmentViewModel.java

```diff
@@ -0,0 +1,81 @@
+package com.radu.newsreader.model.newslist;
+
+import android.annotation.SuppressLint;
+import android.app.Application;
+
+import androidx.annotation.NonNull;
+import androidx.databinding.ObservableArrayList;
+import androidx.databinding.ObservableBoolean;
+import androidx.databinding.ObservableField;
+import androidx.databinding.ObservableList;
+import androidx.lifecycle.AndroidViewModel;
+import androidx.lifecycle.Lifecycle;
+import androidx.lifecycle.LifecycleObserver;
+import androidx.lifecycle.OnLifecycleEvent;
+
+import com.radu.data.NewsRepository;
+import com.radu.newsreader.R;
+import com.radu.newsreader.mapper.ArticleToVMListMapper;
+import com.radu.newsreader.model.article.ArticleItemViewModel;
+import com.radu.newsreader.reactive.SingleLiveEvent;
+
+import java.util.List;
+
+import io.reactivex.android.schedulers.AndroidSchedulers;
+
+public class NewsListFragmentViewModel extends AndroidViewModel implements LifecycleObserver {
```

Comment:
This naming could be improved, as imagine that at some point you would use this view model inside an activity, then you shouldn't need to change to naming because it's used in a different part of the app. Let's remove the 'Fragment' bit.